docs/installation.md

```diff
@@ -49,6 +49,29 @@ Kompose is in Fedora 24, 25 and 26 repositories. You can install it just like an
 sudo dnf -y install kompose
 ```
 
+#### openSUSE/SLE
```

Comment:
Could you move this section down below Windows? I'd like to keep the most popular installation methods higher up!

Otherwise, this LGTM :+1: